core/nandtool/nandtool.c

```diff
@@ -159,28 +182,37 @@ static void nandtool_help(const char *prog)
 	printf("Usage: %s [options] <device>\n", prog);
 	printf("\t-e <start[:size]> - erase size block(s) starting from start\n");
 	printf("\t                    (skip size to erase one block or pass size=0 to erase whole device)\n");
+	printf("\t-j                - write jffs2 cleanmarkers (valid only with erase operation)\n");
+	printf("\t-c                - check device for bad blocks and print summary\n");
+	printf("\n");
 	printf("\t-i <path>         - path of the file to flash (requires -s option)\n");
 	printf("\t-r                - flash raw data\n");
 	printf("\t-s <block>        - start flashing from given block (requires -i)\n");
-	printf("\t-c                - check device for bad blocks and print summary\n");
 	printf("\t-h                - print this help message\n");
 }
 
 
 int main(int argc, char **argv)
 {
-	int check = 0, raw = 0, flash_start = -1, erase_start = -1, erase_size = -1;
+	int check = 0, raw = 0, flash_start = -1, erase_start = -1, erase_size = -1, write_cleanmarkers = 0;
```

Comment:
Nitpick/coding style - use camelCase

Reply:
I would keep the current coding style of the code for readability, full-code renaming can be done in separate PR by the original author @Maxez

Reply:
Sure, I'm going to rewrite nandtool in the future anyway (using `libstorage` interface) and I'll correct coding style to camel case.